ios/FluentUI.Demo/FluentUI.Demo/DemoColorThemes.swift

```diff
@@ -22,82 +22,190 @@ enum DemoColorTheme: CaseIterable {
         }
     }
 
-    var provider: ColorProviding? {
+    var provider2: ColorProviding2? {
```

Comment:
Curious when we are appending 2 to everything? Shouldn't we just plug `ColorProviding2` and leave everything else as is naming-wise?

Reply:
I will remove it! I had them side-by-side to keep things building and obvious while I was replacing the functionality, so it's a side effect of that.